src/client/lib/hooks/useRefTracking.ts

```diff
@@ -0,0 +1,12 @@
+// react hook for getting the `ref` and `refViewId` for tracking
+// `ref` is the current url of the page we want to track
+// `refViewId` is the current ophan page view id we want to track
+export const useRefTracking = (): [string | undefined, string | undefined] => {
```

Comment:
if this returned an object instead of an array it might be easier for the user to know that the hook returns `ref` and `refViewId` at the point of destructuring the return val